src/components/BodyComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState , useEffect} from 'react'
+import CardComponent from './CardComponent'
+import SearchBar from './SearchBar'
+import {Link} from 'react-router-dom'
+const BodyComponent = () => {
+   const [searchData, setsearchData] = useState([])
+   useEffect(()=>{
+    fetchTeamData()
+   },[])
+   async function fetchTeamData(){
+      const teamData = ['nam334','BhaveshSSingh','gk9848970']
+      const arr = []
```

Comment:
arr is getting updated. it is a const and still not throwing error because it is an array. better to use let here.

Reply:
good catch, thanks!